src/haz3lcore/lang/term/IdTagged.re

```diff
@@ -14,6 +14,11 @@ type t('a) = {
   term: 'a,
 };
 
+// To be used if you want to remove the id from the debug output
+// let pp: ((Format.formatter, 'a) => unit, Format.formatter, t('a)) => unit =
+//   (fmt_a, formatter, ta) => {
+//     fmt_a(formatter, ta.term);
+//   };
```

Comment:
I'm open to better ideas on how to enable this. Without it it makes test output difficult to read.